lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -54,67 +55,98 @@ export class Vulnerabilities {
     config: RenovateConfig,
     packageFiles: Record<string, PackageFile[]>
   ): Promise<void> {
+    const vulnerabilityGroups = await this.fetchVulnerabilityGroups(
```

Comment:
For consistency with the now changed interface name, would you please also adapt related naming?